contracts/Example3.sol

```diff
@@ -1,14 +1,12 @@
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
     function getOwnerMargin(uint256 amount)
         public
-        view
+        pure
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        amountForSender = (amount * (100 - 30)) / 100;
```

Comment:
It is better to have a variable when using it multiple times. Here we can have `uint256 marginPercentage=30;` declared inside the function itself so it is read from memory each time